### DISCUSSION TITLE
openDTU mit Venus OS (aber ohne Multiplus)

### DISCUSSION THREAD

**Discussion by Gelegenheitsbastler** [tbnobody/OpenDTU#2080] [Q&A]
Ich möchte die Daten aus der openDTU mit Venus OS auslesen und die Wechselrichterleistung dann in der VRM Übersicht anzeigen. 
Venus OS läuft auf einem Raspberry Pi, die openDTU ist mit einem Wecheselrichter verbunden. Die Wechselrichterdaten werden im Display und im UI der openDTU korrekt angezeigt. 

Ebenfalls sind die Daten in der Geräteliste der Remote-Konsole zu sehen, werden aber nicht als Wechselrichter in der Grafik angezeigt. 

Einen Akku gibt es nicht. 
Die Zählerdaten werden durch einen Shelly 3EM direkt nach dem Zähler erfasst und korrekt angezeigt. Der Wechselrichter ist an L1 angeschlossen. Den Shelly 3EM habe ich durch ein Script zum Venus OS hinzu gefügt. Die openDTU ebenfalls.

Habe ich irgend eine Einstellung falsch gemacht oder etwas vergessen?

![Geräteliste](https://github.com/tbnobody/OpenDTU/assets/33948625/dba0fee6-a17d-4d3c-a354-8477cd5ca19a)

![Remote-Konsole](https://github.com/tbnobody/OpenDTU/assets/33948625/e9b2567e-5b95-4caa-aef1-d5efa162c733)

![VRM](https://github.com/tbnobody/OpenDTU/assets/33948625/15196121-84e4-4244-9c1e-e0e362acb40f)

Für Tipps und Ideen wäre ich dankbar.

**Comment by ms49434** [comment]
Ich habe OpenDTU über dbus-opendtu integriert:
![image](https://github.com/tbnobody/OpenDTU/assets/94918347/41d1643f-0052-4813-ae0c-f822d4217325)
https://github.com/henne49/dbus-opendtu
Alle Wechselrichter werden auch im Menu angezeigt:
![image](https://github.com/tbnobody/OpenDTU/assets/94918347/ad1a522a-6858-4dbf-b5d5-52a17d8bfe91)

**Comment by Gelegenheitsbastler** [comment]
Das habe ich maximal unglücklich formuliert und hätte besser den (gleichen) Link dazu geschrieben.
Ich habe es genau so gemacht. Das meinte ich mit Scripten. 
Allerdings habe ich in dieser Installation keinen Multiplus, sondern aktuell nur den Shelly 3EM und openDTU mit dem Wechselrichter. Sozusagen ein klassisches Balkonkraftwerk dass ich visualisieren möchte.

**Reply by ms49434** [reply]
hast Du in der config.ini von dbus-opendtu beim Inverter als Servicename com.victronenergy.pvinverter und Phase und AcPosition definiert?

**Comment by Gelegenheitsbastler** [comment]
Hier der entsprechende Auszug:

```
.
.
.
#IP of Device to query
Host=192.....

HTTPTimeout=2.5

# Username/Password leave empty if no authentication is required
Username = xy
Password = yz

### Only needed for OpenDTU and ahoy
# AcPosition 0=AC input 1; 1=AC output; 2=AC output 2
# 1st inverter
[INVERTER0]
Phase=L1
DeviceInstance=70
AcPosition=0
#Servicename=com.victronenergy.charger

# 2nd inverter
[INVERTER1]
Phase=L2
.
.
.

Muss das hier aktiviert werden?

#Servicename=com.victronenergy.charger


```

**Reply by ms49434** [reply]
das # rausnehmen und den Servicenamen ändern auf com.victronenergy.pvinverter:

z.B.
[INVERTER0]
Phase=L3
DeviceInstance=34
AcPosition=0
Servicename=com.victronenergy.pvinverter

**Comment by Gelegenheitsbastler** [comment]
Habe ich gemacht. 
Ich erhalte keine Fehlermeldung aber auch nach Reboot des Raspberry Pi (Venus OS) keine Änderung in der Remote Konsole :-(
```

### Only needed for OpenDTU and ahoy
# AcPosition 0=AC input 1; 1=AC output; 2=AC output 2
# 1st inverter
[INVERTER0]
Phase=L1
DeviceInstance=70
AcPosition=0
Servicename=com.victronenergy.pvinverter


```

Muss ich in den Einstellungen noch irgendwo irgend etwas einstellen, damit ein Wechselrichter über die openDTU angezeigt wird?

**Reply by ms49434** [reply]
was sagen denn dbus-spy und svstat /service/dbus-opendtu ?
